src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -423,7 +436,7 @@ def export_config(cmd,
                                         naming_convention=naming_convention)
     elif destination == 'appconfig':
         __write_kv_and_features_to_config_store(dest_azconfig_client, key_values=src_kvs, features=src_features,
-                                                label=dest_label, preserve_labels=preserve_labels,
+                                                label=dest_label, tags=dest_tags, preserve_labels=preserve_labels,
```

Comment:
If `dest_tags` is not provided, this would end up clearing existing tags instead of preserving them.